packages/@uppy/core/src/index.js

```diff
@@ -1076,6 +1076,7 @@ class Uppy {
       const currentProgress = this.getFile(file.id).progress
       this.setFileState(file.id, {
         progress: Object.assign({}, currentProgress, {
+          postprocess: this.postProcessors.length > 0,
```

Comment:
`postprocess` is normally an object, not a boolean, so we should prob do something like:

```suggestion
          postprocess: this.postProcessors.length > 0 ? {
            mode: 'indeterminate'
          } : null,
```